__snapshots__/node-workspace.js

```diff
@@ -1747,6 +1687,125 @@ exports['NodeWorkspaceDependencyUpdates run prefers release-as configuration ove
   ]
 ]
 
+exports['NodeWorkspaceDependencyUpdates run updates dependency to pre-release version changes'] = `
```

Comment:
The upstream diff is really confusing, so I'm commenting on the commit itself.

@joeldodge79 this is the breaking behavior that might be the most confusing and I'd like to get your take. What's happening with "always-link-local" default to `true`, the pre-release versions are being bumped into non-pre-release packages and those are getting a patch bump. Do you agree with this behavior?

Reply:
Hm, this piece seems less appealing on the surface. But again, at least given how we currently build/publish our packages from looker-open-source/sdk-codegen we'd fail to find `@here/pkgA 1.1.1` since it was bumped to `1.1.2-alpha.0` so we'd need this change. It could be we're doing something wrong with our build/publish where it should still be able to use the previous version even though there's no longer a local representation of it. 

anyway, I think I'm still good with this change!